lib/portfinder.js

```diff
@@ -88,6 +92,22 @@ exports.getPort = function (options, callback) {
   if (!callback) {
     callback = options;
     options = {};
+
+  }
+
+  options.port   = Number(options.port) || Number(exports.basePort);
+  options.host   = options.host    || null;
+  options.stopPort = Number(options.stopPort) || Number(exports.highestPort);
+
+  if(!options.startPort) {
+    options.startPort = Number(options.port);
+    if(options.startPort < 1024) {
```

Comment:
https://github.com/indexzero/node-portfinder/issues/65 filed due to the above line. @Prithvirajbilla do u mind making a quick pr - or i can but starting work in like.. now so would be in a few hours

Reply:
https://github.com/indexzero/node-portfinder/pull/66

Reply:
I saw your PR. Thanks for getting to this quickly.  I will clean this up and will add a test for this special case (port 80) when I get time.

Reply:
LOL - yeah you'd be surprised how many upstream libs use this thing :) no worries, happens.